app/templates/macros/toolkit_forms.html

```diff
@@ -209,3 +209,38 @@
     </fieldset>
   {% endif %}
 {%- endmacro %}
+
+{% macro multiquestion(question_content, service_data, errors, question_number=None, get_question=None) -%}
+<div class="question" id="{{ question_content.id }}">
+  <label for="input-{{ question_content.id }}">
+      <span class="question-heading{% if question_content.hint %}-with-hint{% endif %}">
+          <span class="question-number">
+            {{ question_number }}
+          </span>
+        
+        {{ question_content.question }}
+        
+      </span>
+
+  </label>
+  <legend>
+    <span class="visually-hidden">{{ question_content.name }}</span>
+  </legend>
+  {% if question_content.question_advice %}
+        <span class="question-advice" id="input-{{ name }}-question-advice">
+          {{ question_content.question_advice|markdown }}
+        </span>
+  {% endif %}
+  {% if question_content.hint %}
+  <p class="question-description">{{ question_content.hint|markdown }}</p>
+  {% endif %}
```

Comment:
I just copied it from `boolean_list` - I'll change it both places if you reckon that will work?  To be honest we don't have any hints on our sole multiquestion, so the point is moot.